### ISSUE TITLE
preload() has different parsing rules from other function calls

### ISSUE THREAD

**Issue by Razoric480** [godotengine/godot#52499]
### Godot version

3.3.stable

### System information

Windows 10

### Issue description

`preload()` in GDScript runs on a different ruleset than the usual parser and does not support newlines. When formatting a call to preload like:
```gdscript
const MinimapTileset := preload("res://addons/godot-metroidvania-kit/Minimap/Tileset/minimap_tileset.tres")
```
into:
```gdscript
const MinimapTileset := preload(
	"res://addons/godot-metroidvania-kit/Minimap/Tileset/minimap_tileset.tres"
)
```
it will report a parsing error about a misplaced \\n, whereas any other function call support it just fine.
```gdscript
onready var has_foyer := _has(
	"foyer"
)
```

### Steps to reproduce

1. New project
2. Add Node
3. Add script to Node
4. Paste:
```gdscript
extends Node

const default_env_bad := preload(
	"res://default_env.tres"
)

onready var default_env_okay := load(
	"res://default_env.tres"
)
```

**Comment by Razoric480** [comment]
This seems to be already fixed in 4.0, 
```gdscript
extends Node

const default_env_bad := preload(
	"res://default_env.tres"
)

@onready
var default_env_okay := load(
	"res://default_env.tres"
)
```
so it's quite possible that fixing this for 3.x will not be worth the time.

**Comment by kdiduk** [comment]
I can take care of this issue. I've investigated it a little bit.
You can assign this ticket to me.

**Comment by Calinou** [comment]
> I can take care of this issue. I've investigated it a little bit.
> You can assign this ticket to me.

Feel free to work on this, but we can't assign issues to people who aren't core contributors.

**Comment by kdiduk** [comment]
I've made a Pull-Request #52521 with a fix of this issue. Please, review (there's only 1 commit with few lines :blush: )

**Comment by kdiduk** [comment]
I believe this ticket can be already closed. This was even mentioned in the changes of 3.4 beta 6 (https://godotengine.org/article/dev-snapshot-godot-3-4-beta-6)

**Comment by akien-mga** [comment]
Fixed by #52521.
